server/sandbox_run.go

```diff
@@ -80,8 +84,52 @@ func (s *Server) trustedSandbox(req *pb.RunPodSandboxRequest) bool {
 }
 
 func (s *Server) runContainer(container *oci.Container, cgroupParent string) error {
-	if err := s.Runtime().CreateContainer(container, cgroupParent); err != nil {
-		return err
+	intermediateMountPoint := container.IntermediateMountPoint()
+
+	if intermediateMountPoint == "" {
+		if err := s.Runtime().CreateContainer(container, cgroupParent); err != nil {
+			return err
+		}
+	} else {
+		var wg sync.WaitGroup
+		wg.Add(1)
+		go func() {
```

Comment:
Should this be in a common function, so ti could eventually be shared between buildah/podman/cri-o?